src/systems/camera-system.js

```diff
@@ -290,6 +290,7 @@ export class CameraSystem {
       this.pivot,
       distanceMod || 1
     );
+    this.ensureListenerIsParentedCorrectly(scene);
```

Comment:
In this case `AudioListener.updateMatrixWorld` will not be called in `moveRigSoCameraLooksAtPivot`, so we could do it in either order. To me though, it seems more correct to call `ensureListenerIsParentedCorrectly` _before_ `moveRigSoCameraLooksAtPivot`. 

In its current order, we move the camera (the `AudioListener`'s parent) close to the object, then reparent the `AudioListener` so that it remains close to the avatar. It seems like an accident that `AudioListener.updateMatrixWorld` wasn't called in `moveRigSoCameraLooksAtPivot`. If it were, then we would still be experiencing the blip.

For this reason, I recommend calling `ensureListenerIsParentedCorrectly` _before_ `moveRigSoCameraLooksAtPivot` and also _before_ `orbit` (in `tick`)

Reply:
OK, your suggestion sounds safer. I'll update.

Reply:
Done